app/src/features/surveys/view/SurveyObservations.tsx

```diff
@@ -106,7 +106,7 @@ export enum SUBMISSION_MESSAGE_TYPE {
   'FAILED_PREP_DWC_ARCHIVE' = 'Failed to prep DarwinCore Archive',
   'FAILED_PREP_XLSX' = 'Failed to prep XLSX',
   'FAILED_PERSIST_PARSE_ERRORS' = 'Failed to persist parse errors',
-  'FAILED_GET_VALIDATION_RULES' = 'Failed to get validation rules',
+  'FAILED_GET_VALIDATION_RULES' = 'Failed to get validation rules.  It is possible that your focal species is incorrect.',
```

Comment:
I don't know how I feel about this addition. This enum is just supposed to store error message types; I think suggestions such as "fix your focal species" should belong in the actual error message, not in the message type string - especially given that the phrasing here is also a bit subjective. Moreover, if we did want to change this enum value, we would need a corresponding migration as well.

Reply:
Yeah makes sense.

I do think it adds additional information which is good, but it also could be misleading if the focal species isnt the issue (and you submitted the wrong template or something). The app side needs an update in general around how better to display error messages (rolling them up, having helper icons for more details, etc). So for now we can just leave it as it was, and a user may just need to reach out to Robin & Co for assistance.